### DISCUSSION TITLE
Bootstrap - [InstallFolder] should not be empty string by default

### DISCUSSION THREAD

**Discussion by SavageStyle** [wixtoolset/issues#6397] [Questions]
[InstallFolder] default value should be [ProgramFilesFolder ] as it is used in case [InstallFolder] is empty.
This line of code is only valid if used specified [InstallFolder] manually:
`<Variable Name="LaunchTarget" Value="[InstallFolder]\MyApp.exe" />`

Is there any workaround for this issue right now?

**Comment by SavageStyle** [comment]
Seems workaround is simply to set up this manually:
```
<Variable
	Name='InstallFolder'
	Type='string'
	Value='[ProgramFilesFolder]' />
```
But what about x64 apps?

**Reply by rseanhall** [reply]
This isn't a workaround, it's the way to do it. Make the bundle own all configuration.

You might be interested in `ProgramFiles6432Folder` (https://wixtoolset.org/documentation/manual/v3/bundle/bundle_built_in_variables.html).
